app/controllers/courses_controller.rb

```diff
@@ -5,7 +5,7 @@
 
 class CoursesController < ApplicationController
   before_action :set_organization
-  before_action :set_course, only: [:show, :refresh, :manage_deadlines, :save_deadlines, :enable, :disable, :manage_unlocks, :save_unlocks]
+  before_action :set_course, only: [:show, :edit, :update, :refresh, :manage_deadlines, :save_deadlines, :enable, :disable, :manage_unlocks, :save_unlocks ]
```

Comment:
We will probably rewrite this line at some later pr, when we will have most of controller methods implemented and merged